src/cassandra/statement.rs

```diff
@@ -441,6 +442,17 @@ impl Statement {
         unsafe { cass_statement_set_custom_payload(self.0, payload.inner()).to_result(self) }
     }
 
+    /// Sets the statement's tracing flag.
+    ///
+
```

Comment:
Something's gone wrong with the comment formatting here - please can you fix, so we get good docs?